### ISSUE TITLE
Milestones to adding support for another language

### ISSUE THREAD

**Issue by kasisnu** [jdkato/prose#30]
Can we use this issue to document the pieces that will directly be impacted if we need to add support for _any_ other language?

I would be interested in working on this but I don't have *any* domain expertise so I could use all the help.

Thanks

**Comment by remisb** [comment]
Actually I'm interested in adopting it to process another language. It's sad that this feature is left for 3.0 version branch. I guess it will not be even evaluated more seriously in the current year. :( . Of course I can try to make a branch and start doing something about it, but forking sound too much.

**Comment by jdkato** [comment]
@remisb: I expect that 3.0.0 will be released around July of this year (around the same time as 2.0.0). There's a lot that I'd like to improve, including support for other languages.

**Comment by doutuiya** [comment]
any progress ?

**Comment by jdkato** [comment]
No, I've admittedly fallen off schedule and haven't had significant time to dedicate towards 3.0.0 yet.
